openlibrary/macros/QueryCarousel.html

```diff
@@ -9,6 +9,11 @@
 $# * search (bool) -- whether to include search within collection
 $# * layout (str) -- layout type, default 'carousel', currently also supports 'grid'
 
+$ web_lang= get_lang() or 'en'
+$if user_lang_only and web_lang:
+  $ user_lang= convert_iso_to_marc(web_lang)
```

Comment:
Because of the `or 'en'`, this can never be falsy, so we should be able to move this down.
```suggestion
$if user_lang_only:
  $ user_lang= convert_iso_to_marc(get_lang() or 'en')
```